src/Dynatrace.OpenTelemetry.Exporter.Metrics/DynatraceMetricsExtensions.cs

```diff
@@ -27,64 +27,75 @@ internal static class DynatraceMetricsExtensions
 		public static DynatraceMetric ToLongCounterDelta(this Metric metric, MetricPoint metricPoint, ILogger logger)
 			=> DynatraceMetricsFactory.CreateLongCounterDelta(
 				metric.Name,
-				metricPoint.LongValue,
+				metricPoint.GetSumLong(),
 				metricPoint.GetAttributes(logger),
 				metricPoint.EndTime);
 
 		public static DynatraceMetric ToDoubleCounterDelta(this Metric metric, MetricPoint metricPoint, ILogger logger)
 			=> DynatraceMetricsFactory.CreateDoubleCounterDelta(
 				metric.Name,
-				metricPoint.DoubleValue,
+				metricPoint.GetSumDouble(),
 				metricPoint.GetAttributes(logger),
 				metricPoint.EndTime);
 
 		public static DynatraceMetric ToLongGauge(this Metric metric, MetricPoint metricPoint, ILogger logger)
 			=> DynatraceMetricsFactory.CreateLongGauge(
 				metric.Name,
-				metricPoint.LongValue,
+				metricPoint.GetGaugeLastValueLong(),
 				metricPoint.GetAttributes(logger),
 				metricPoint.EndTime);
 
 		public static DynatraceMetric ToDoubleGauge(this Metric metric, MetricPoint metricPoint, ILogger logger)
 			=> DynatraceMetricsFactory.CreateDoubleGauge(
 				metric.Name,
-				metricPoint.DoubleValue,
+				metricPoint.GetGaugeLastValueDouble(),
 				metricPoint.GetAttributes(logger),
 				metricPoint.EndTime);
 
 		public static DynatraceMetric ToDoubleHistogram(this Metric metric, MetricPoint metricPoint, ILogger logger)
 		{
-			var min = GetMinFromBoundaries(metricPoint);
-			var max = GetMaxFromBoundaries(metricPoint);
+			var buckets = GetHistogramBuckets(metricPoint);
+			var min = GetMinFromBoundaries(metricPoint.GetHistogramSum(), metricPoint.GetHistogramCount(), buckets);
+			var max = GetMaxFromBoundaries(metricPoint.GetHistogramSum(), metricPoint.GetHistogramCount(), buckets);
 
 			return DynatraceMetricsFactory.CreateDoubleSummary(
 				metric.Name,
 				min,
 				max,
-				metricPoint.DoubleValue,
-				metricPoint.LongValue,
+				metricPoint.GetHistogramSum(),
+				metricPoint.GetHistogramCount(),
 				metricPoint.GetAttributes(logger),
 				metricPoint.EndTime);
 		}
 
-		private static double GetMinFromBoundaries(MetricPoint pointData)
+		private static List<HistogramBucket> GetHistogramBuckets(MetricPoint pointData)
 		{
-			if (pointData.BucketCounts.Length == 1)
+			var buckets = new List<HistogramBucket>();
+			foreach (var bucket in pointData.GetHistogramBuckets())
 			{
-				// In this case, only one bucket exists: (-Inf, Inf). If there were any boundaries, there
-				// would be more counts.
-				if (pointData.BucketCounts[0] > 0)
+				buckets.Add(bucket);
+			}
+
+			return buckets;
+		}
+
+		private static double GetMinFromBoundaries(double sum, long count, IReadOnlyList<HistogramBucket> buckets)
```

Comment:
`count` is the number of values, right?

Reply:
Correct, I changed the names to `histogramSum` and `histogramValueCount` to better reflect that. :)